src/styles/components/typeahead.less

```diff
@@ -0,0 +1,27 @@
+.typeahead.inverse {
```

Comment:
@MatApple's saying that all of these extended styles only apply for the `.inverse` class. But, I see that the `Typeahead` component uses the same classes as Canvas, so there's no need to override most of those styles.

But I do think we should generically extend the `.clickable` class for `li`s, as well as the styles for the `li.active` declaration, regardless of the `.inverse` modifier.